src/snowflake/cli/_plugins/workspace/manager.py

```diff
@@ -34,9 +36,10 @@ def get_entity(self, entity_id: str):
             return self._entities_cache[entity_id]
         if entity_id not in self._project_definition.entities:
             raise ValueError(f"No such entity ID: {entity_id}")
-        entity_model_cls = self._project_definition.entities[entity_id].__class__
+        entity_model = self._project_definition.entities[entity_id]
```

Comment:
nit: I usually prefer to use `get` and check for `None`. It avoids the double lookup and guards against weird state being introduced elsewhere.

Reply:
Done